Makefile

```diff
@@ -44,14 +44,14 @@ GO_IMPORT_SPACES := ${EXT_IMPORT},\
 GO_IMPORT:=$(subst $(space),,$(GO_IMPORT_SPACES))
 
 
-PHONE: generated-code
+.PHONY: generated-code
 generated-code:
-	PATH=$(DEPSGOBIN):$$PATH protoc -I=. --go_out="${EXT_IMPORT}:." extproto/ext.proto
+	PATH=$(DEPSGOBIN):$$PATH $(DEPSGOBIN)/protoc -I=. -I=./external --go_out="${EXT_IMPORT}:." extproto/ext.proto
```

Comment:
So, this this mean that any locally installed `protoc` is irrelevant?  if so, please delete protoc checks from the local validation script (or at least make it optional)

Reply:
Yea once this is merged, unless there is any other repo related to GME, a local protoc installation would be irrelevant. 

You are referring to altering the validation script in GME correct?